CIPs/0022.md

```diff
@@ -0,0 +1,40 @@
+---
+cip: 0022
```

Comment:
Just to note that we will be changing the CIP number as only editors can assign a number to a CIP